tests/test_bindings.py

```diff
@@ -1546,3 +1551,84 @@ def test_match_statement_bindings_as():
         """
     )
     assert list(get_bindings(node)) == ["b", "c"]
+
+
+@exception_group
+def test_try_star_no_binding():
+    """
+    ..code:: python
+
+        TryStar(
+            stmt* body,
+            excepthandler* handlers,
+            stmt* orelse,
+            stmt* finalbody,
+        )
+    """
+    node = _parse(
+        """
+        try:
+            a = something_stupid()
+        except* ExceptionGroup:
+            b = recover()
+        else:
+            c = otherwise()
+        finally:
+            d = finish()
+        """
+    )
+    assert list(get_bindings(node)) == ["a", "b", "c", "d"]
+
+
+@exception_group
+def test_try_star_binding():
+    """
+    ..code:: python
+
+        TryStar(
+            stmt* body,
+            excepthandler* handlers,
+            stmt* orelse,
+            stmt* finalbody,
+        )
+    """
+    node = _parse(
+        """
+        try:
+            a = something_stupid()
+        except* ExceptionGroup as exc:
+            b = recover()
+        else:
+            c = otherwise()
+        finally:
+            d = finish()
+        """
+    )
+    assert list(get_bindings(node)) == ["a", "exc", "b", "c", "d"]
+
+
+@exception_group
+def test_try_star_binding_walrus_exeption_type():
+    """
+    ..code:: python
+
+        TryStar(
+            stmt* body,
+            excepthandler* handlers,
+            stmt* orelse,
+            stmt* finalbody,
+        )
+    """
+    node = _parse(
+        """
+        try:
+            a = something_stupid()
+        except* (grp_type := ExceptionGroup) as grp:
```

Comment:
What madness is this!  I love some of the possibilities enabled by trying to keep syntax uniform.

Reply:
It's really impressive what possibilities there.

But I was honestly surprised that something like
```python
try:
    pass
except* ExceptionGroup as (first, *_):
    pass
```
does not work